include/seqan3/core/simd/view_to_simd.hpp

```diff
@@ -15,7 +15,7 @@
 #include <seqan3/alphabet/concept.hpp>
 #include <seqan3/core/detail/pack_algorithm.hpp>
 #include <seqan3/utility/simd/concept.hpp>
-#include <seqan3/core/simd/simd_algorithm.hpp>
+#include <seqan3/utility/simd/simd_algorithm.hpp>
```

Comment:
Side question: why prefix the header with `simd_` if it is already in the simd submodule? why not just `algorithm.hpp`?

Reply:
not sure the list said:

* simd/simd_algorithm.hpp -> utility/simd/.
* simd/simd_traits.hpp -> utility/simd/.

so I kept it, I added this question to https://github.com/seqan/product_backlog/issues/160

Reply:
I think it would be ok to rename them without the prefix. That would be more consistent with the rest. I think the configs in alignment need to do this as well.

Reply:
Resolution 2020-11-23:

We should rename them right away.

Reply:
I'm not in favour for that; The class is called `simd_traits` and the header reflects that name.

Reply:
Ah ok. well if the main class in this actually called `simd_traits` I would also keep it as is.
What about `simd_algorithm?` is it the same case? Otherwise we decided to call it just `algorithm.hpp`